packages/x-components/src/views/design-system/design-system-tag.vue

```diff
@@ -143,15 +143,38 @@
         <span>text tag extra large name</span>
       </div>
     </article>
+    <article class="x-list x-list--wrap x-list--gap-06 x-list--align-start">
+      <h2 class="x-title2">Curated</h2>
+      <div class="x-tag x-tag--is-curated">
+        <CuratedCheck />
+        <span>text tag</span>
+      </div>
+      <div class="x-tag x-tag--is-curated">
```

Comment:
The curated tags work with pill and card variants, but do not work with ghost variant

Reply:
Now the ghost variant removes the background in curated RTs, but when selected it does not apply the font weight, which causes them to look identical when selected and when un-selected. You can see so in the second RT of this picture which has the ghost variant applied

![image](https://user-images.githubusercontent.com/68222542/151928083-5baf8d4d-be5c-43a0-8bd7-409d5e34cc65.png)